page.py

```diff
@@ -205,20 +205,20 @@ def new_target(self):
 
     def _play_target_sound(self):
         if self._activity.mode in ['letter', 'find by letter']:
-            play_audio_from_file(os.path.join(
+            aplay.play(os.path.join(
                     self._sounds_path,
-                    self._media_data[self.target][1]))
+                    self._media_data[self.target][1])[:-1])
         elif self._activity.mode == 'picture':
-            play_audio_from_file(os.path.join(
+            aplay.play(os.path.join(
                     self._sounds_path,
-                    self._media_data[self.target][1]))
-            GLib.timeout_add(1000, play_audio_from_file, os.path.join(
+                    self._media_data[self.target][1])[:-1])
+            GLib.timeout_add(1000, aplay.play, os.path.join(
                     self._sounds_path,
-                    self._media_data[self.target][0]))
+                    self._media_data[self.target][0])[:-1])
         else:
-            play_audio_from_file(os.path.join(
+            aplay.play(os.path.join(
                     self._sounds_path,
-                    self._media_data[self.target][0]))
+                    self._media_data[self.target][0])[:-1])
```

Comment:
Can you explain the need for `[:-1]` slicing?

Currently, I am at this error:
```
1608878628.474940 ERROR root: gst-resource-error-quark: Resource not found. (3) ../gstreamer/plugins/elements/gstfilesrc.c(536): gst_file_src_start (): /GstPlayBin:playbin/GstURIDecodeBin:uridecodebin6/GstFileSrc:source:
No such file '/xxx/git/sugar/activities/iknowmyabcs/sounds/es/dado.og'
```

Reply:
I've made some changes. Please recheck

Reply:
Re-reviewed